docs/docsite/Makefile

```diff
@@ -1,7 +1,7 @@
 SITELIB = $(shell python -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")
 FORMATTER=../../hacking/module_formatter.py
 DUMPER=../../hacking/dump_playbook_attributes.py
-CPUS ?= 1
+CPUS := $(shell nproc)
```

Comment:
see this for a OS compatible way to do this https://github.com/ansible/ansible/blob/devel/Makefile#L49